src/abstractdataframe/abstractdataframe.jl

```diff
@@ -106,18 +106,18 @@ Set column names
 names!(df::AbstractDataFrame, vals)
 ```
 
-### Arguments
+**Arguments**
```

Comment:
Indeed. Maybe file a bug, as this doesn't sound like a useful behavior in general?